rpgJavaInterpreter-core/build.gradle

```diff
@@ -59,12 +59,12 @@ generateGrammarSource {
     arguments += ['-package', 'com.smeup.rpgparser']
     outputDirectory = file(generatedMain + "/com/smeup/rpgparser")
 }
-compileKotlin.dependsOn generateGrammarSource
 compileJava.dependsOn generateGrammarSource
 compileJava.source generatedMainFile, sourceSets.main.java
-compileKotlin.source generatedMainFile, sourceSets.main.java, sourceSets.main.kotlin
 compileKotlin {
+    source generatedMainFile, sourceSets.main.java, sourceSets.main.kotlin
     kotlinOptions.freeCompilerArgs += ["-Xuse-experimental=kotlin.ExperimentalUnsignedTypes"]
+    dependsOn generateGrammarSource
```

Comment:
It makes sense. Personally I prefer to have the dependsOn clauses outside the single tasks because to me it makes more evident the relations between tasks, but it is a matter of test and this make the script cleaner imho